docs/usage/self-hosted-configuration.md

```diff
@@ -98,6 +98,14 @@ Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer pack
 
 This configuration will be applied after all other environment variables so that it can be used to override defaults.
 
+## dockerChildPrefix
+
+Adds a custom prefix to the default Renovate sidecar Docker containers name and label.
+
+If this is set to `myprefix_` the final image name for `renovate/node` would be named `myprefix_renovate_node` instead of currently used `renovate_node` and be labeled `myprefix_child` instead of `renovate_child`.
```

Comment:
```suggestion
If this is set to `myprefix_` the final image name for `renovate/node` would be named `myprefix_node` instead of currently used `renovate_node` and be labeled `myprefix_child` instead of `renovate_child`.
```